src/features/header/Header.js

```diff
@@ -1,11 +1,14 @@
 import './Header.css'
+import { Link } from 'react-router-dom'
 
 function Header() {
   return(
     <header>
-      <h1 className='header-logo'>Mars Chronicles</h1>
+      <Link to="/">
+        <h1 className='header-logo'>Mars Chronicles</h1>
+      </Link>
```

Comment:
Love that you've made the header a a link to the homepage - good consideration of making the user experience intuitive